scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -144,12 +145,19 @@ impl ReplicatorConsumer {
         values_for_update.extend(values.iter());
 
         for column_name in &self.non_key_columns {
-            match self.table_schema.columns.get(column_name).unwrap().type_ {
+            match self
+                .table_schema
+                .columns
+                .get(column_name)
+                .unwrap()
+                .type_
+                .clone()
```

Comment:
No need to introduce clone here. You can put `&` before `self.table_schema...` and the type will be matched on by reference.